db/migrate/20220110085736_create_reservations.rb

```diff
@@ -0,0 +1,14 @@
+class CreateReservations < ActiveRecord::Migration[7.0]
+  def change
+    create_table :reservations do |t|
+      t.belongs_to :seance
+      t.belongs_to :ticketdesk
+      t.belongs_to :client
```

Comment:
Your application is a couple lines of code. This application uses **database** to store data somewhere. 

When you add associations here - in migration - you add [FOREIGN KEYS](https://www.w3schools.com/sql/sql_foreignkey.asp) and Indexes to the **DATABASE** but your application is not aware of this change cause migration is related only to **DATABASE**.

When you adding associations in migration, you **HAVE TO** always add association in **MODEL**. Why? Cause **MODEL** should be always aware of associations defined on **DATABASE** level.

Related article - https://guides.rubyonrails.org/association_basics.html

Reply:
Oh did not know that one. Thank you for that.